contracts/bonding/BondingManager.sol

```diff
@@ -127,20 +116,10 @@ contract BondingManager is IBondingManager, Manager {
         _;
     }
 
-    /*
-     * @dev BondingManager constructor. Sets a pre-existing address for the LivepeerToken contract
-     * @param _token LivepeerToken contract address
-     */
-    function BondingManager(
-        address _registry,
-        address _token,
-        uint256 _numActiveTranscoders,
-        uint64 _unbondingPeriod
-    )
-        Manager(_registry)
-    {
-        // Set LivepeerToken address
-        token = LivepeerToken(_token);
+    function BondingManager(address _controller) Manager(_controller) {}
+
+    function initialize(uint64 _unbondingPeriod, uint256 _numActiveTranscoders) external beforeInitialization returns (bool) {
+        finishInitialization();
```

Comment:
It's important to call this to set the `initialized` flag (inherited from `Initializable`) to `true` so that the `initialize` method cannot be called more than once